app/src/main/java/io/scalac/degree/android/fragment/SpeakersFragment.java

```diff
@@ -149,23 +311,33 @@ public View getView(final int position, View convertView, ViewGroup parent) {
             final ViewHolder holder;
 
             if (convertView == null) {
-                viewItem = getActivity().getLayoutInflater().inflate(R.layout.speakers_all_list_item, parent, false);
+                viewItem = getActivity().getLayoutInflater()
```

Comment:
wiem ze "nie ja to pisalem", ale ta metoda jest zdecydowanie za dluga :) przynajmniej na dwie to rozbij